client/src/actions/index.js

```diff
@@ -197,6 +197,7 @@ const requestDifferentialExpression = (set1, set2, num_genes = 50) => async (
         }),
         body: JSON.stringify({
           mode: "topN",
+          two_lists: true,
```

Comment:
Is this used?

Reply:
Backend accepts it to toggle on the two list formatted response

Reply:
This would seem to encode `diffExp` as a magic word, but a bit more cryptically, is there a reason we aren't using something more explicit like `isDifferentialExpression`?

Reply:
I'm lost, could quote the lines you're referencing?

Reply:
To rephrase, line 200 uses the variable name `two_lists`, but does this variable name really mean `isDifferentialExpression`?

Reply:
No it means, we want the backend to return the diffexp results as an object with two lists, rather than a single list

Reply:
Are there any other circumstances we want the system to do this other than diffexp?

Reply:
No, this is the only endpoint that accepts this argument